providers/amazon/src/airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -57,12 +57,16 @@
     AirflowNotFoundException,
     AirflowProviderDeprecationWarning,
 )
-from airflow.hooks.base import BaseHook
 from airflow.providers.amazon.aws.utils.connection_wrapper import AwsConnectionWrapper
 from airflow.providers.amazon.aws.utils.identifiers import generate_uuid
 from airflow.providers.amazon.aws.utils.suppress import return_on_error
 from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
 from airflow.providers_manager import ProvidersManager
+
+try:
+    from airflow.sdk import BaseHook
+except ImportError:
+    from airflow.hooks.base import BaseHook  # type: ignore
```

Comment:
Worth moving to version_compat.py -- same as https://github.com/apache/airflow/pull/52528

Reply:
Will investigate and handle here: https://github.com/apache/airflow/issues/52676